src/goto-programs/goto_program.cpp

```diff
@@ -32,51 +37,80 @@ std::ostream& goto_programt::output_instruction(
   std::ostream& out,
   instructionst::const_iterator it) const
 {
-  out << "        // " << it->location_number << " ";
+  return output_instruction(ns, identifier, out, *it);
```

Comment:
Should this function just be removed? Isn't there a danger by passing in the iterator the iterator could actually be advanced which would be somewhat surprising to the callee. Passing the actual instruction ensures this won't happen.